catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/IOR.java

```diff
@@ -1,3 +1,18 @@
+/*
+ *  Copyright (c) Codice Foundation
+ *  <p>
+ *  This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ *  General Public License as published by the Free Software Foundation, either version 3 of the
+ *  License, or any later version.
+ *  <p>
+ *  This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
```

Comment:
That should have been kept the same as it was and not replaced with the Codice one.